.github/workflows/ArduinoBuild.yml

```diff
@@ -0,0 +1,49 @@
+# ArduinoBuild.yml
+#
+# Github workflow script that compiles all examples from the OpenMRNLite
+# Arduino library.
+#
+# Copyright (C) 2021  Balazs Racz
+#
+
+# Name of workflow
+name: ArduinoBuild
+on:
+  push:
+    paths:
+      - '**.ino'
+      - '**rduino**'
+      - '**ArduinoBuild.yml'
+  pull_request:
+jobs:
+  build:
+    name: Build ESP32 examples
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Generate OpenMRNLite library
+      run: |
+        mkdir --parents $HOME/Arduino/libraries/OpenMRNLite
+        $GITHUB_WORKSPACE/arduino/libify.sh $HOME/Arduino/libraries/OpenMRNLite $GITHUB_WORKSPACE -f -l
+        rm -f $GITHUB_WORKSPACE/arduino/examples/Stm*/build_opt.h
+
+    - name: Compile all STM32 examples
+      uses: ArminJo/arduino-test-compile@v3.0.0
+      with:
+        platform-url: https://raw.githubusercontent.com/stm32duino/BoardManagerFiles/master/STM32/package_stm_index.json
+        arduino-board-fqbn: STM32:stm32:Nucleo_144:pnum=NUCLEO_F767ZI,upload_method=MassStorage,xserial=generic,usb=CDCgen,xusb=FS,opt=osstd,rtlib=nano
+        sketch-names: Stm32*.ino
+        build-properties: '{ "All": "-DHAL_CAN_MODULE_ENABLED" }'
+        debug-compile: true
+        
+    - name: Compile all ESP32 examples
+      uses: ArminJo/arduino-test-compile@v3.0.0
+      with:
+        platform-url: https://dl.espressif.com/dl/package_esp32_index.json
```

Comment:
Take a look at the most recent 1.0.6 release as the json file moved to GitHub IIRC.

Reply:
done